app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  validates :body, presence: true, length: { maximum: 80_000 }
+  validates :user_id, presence: true
+  validates :post_id, presence: true
```

Comment:
Guys, you don't need to validate this. The relationship between models has an implicit validation of presence. Check this: https://guides.rubyonrails.org/active_record_validations.html#presence.